### ISSUE TITLE
Assertion failed: err == noErr (again)

### ISSUE THREAD

**Issue by peaxels** [rsjaffe/MIDI2LR#76]
Using OS X 10.11.1 and MIDI2LR 0.7.0, I get the error

    Assertion failed: err == noErr

when trying to start MIDI2LR.

A similar issue was reported [here](https://github.com/rsjaffe/MIDI2LR/issues/65), although the workaround does not apply any more.

---

**Solution / Workaround:**
The file `MIDI2LR.lrplugin/MIDI2LR.app/Contents/MacOS/MIDI2LR` was not marked as executable. A
``` bash
chmod +x "MIDI2LR.lrplugin/MIDI2LR.app/Contents/MacOS/MIDI2LR"
```
fixed it.

**Comment by rsjaffe** [comment]
:anguished: I'll fix that when I have the opportunity. Executable flag apparently not carried over with my current work flow.

**Comment by rsjaffe** [comment]
fixed in 0.7.0 release package. not fixed in the 0.7.0a test package.
